src/sumo_store_mysql.erl

```diff
@@ -276,6 +273,36 @@ find_by(DocName, Conditions, SortFields, Limit, Offset, State) ->
       evaluate_execute_result(Error, State)
   end.
 
+-spec fetch(DocName, Id, State) -> Response when
+  DocName  :: sumo:schema_name(),
+  Id       :: sumo:field_value(),
+  State    :: state(),
+  Response :: sumo_store:result(sumo_internal:doc(), state()).
+fetch(DocName, Id, State) ->
+  IdFieldName = sumo_internal:id_field_name(DocName),
+  case find_by(DocName, [{IdFieldName, Id}], [], 1, 0, State) of
+    {ok, [Doc], _} -> {ok, Doc, State};
+    {ok, [], _}    -> {error, notfound, State};
+    Error          -> Error
+  end.
+
+-spec count(DocName, State) -> Response when
+  DocName  :: sumo:schema_name(),
+  State    :: state(),
+  Response :: sumo_store:result(non_neg_integer(), state()).
+count(DocName, State) ->
+  StatementName = prepare(DocName, count, fun() ->
+    ["SELECT COUNT(*) FROM ", escape(DocName)]
+  end),
+  case execute(StatementName, State) of
+    #result_packet{rows = [Result]} ->
+      %% TODO: check if there is a better way to extract the result count
+      [[_, [Value], _]] = io_lib:format("~w", [Result]),
```

Comment:
Yeap, the thing `emysql` returns an string with an ascii character (which is the the result count) instead of an integer. And I agree, we should find a better way to address this issue, let's wait for @elbrujohalcon comments!